src/neuroconv/datainterfaces/behavior/audio/audiointerface.py

```diff
@@ -70,14 +45,23 @@ def __init__(self, file_paths: list, verbose: bool = False):
                 "The currently supported file format for audio is WAV file. "
                 f"Some of the provided files does not match this format: {format_is_not_supported}."
             )
+
+        self._number_of_audio_files = len(file_paths)
         self.verbose = verbose
         super().__init__(file_paths=file_paths)
+        self._segment_starting_times = None
 
     def get_metadata_schema(self) -> dict:
         metadata_schema = super().get_metadata_schema()
-        time_series_metadata_schema = get_schema_from_hdmf_class(TimeSeries)
-        metadata_schema["properties"]["Behavior"] = get_base_schema(tag="Behavior")
+
+        time_series_metadata_schema_path = (
+            Path(__file__).parent.parent.parent.parent / "schemas" / "time_series_schema.json"
+        )
```

Comment:
Is there a reason we are using this over a standard get_hdmf_schema call?

Reply:
Same reason we're centralizing everything from #14, makes it magnitudes easier to read and debug if we have a file that we can look at without having to generate it code every time